tests/unit/test_context.py

```diff
@@ -98,6 +98,8 @@ async def test_images(assets: pathlib.Path) -> None:
     assert ctx.images["image_a"].dockerfile == LocalPath("dir/Dockerfile")
     assert ctx.images["image_a"].full_dockerfile_path == workspace / "dir/Dockerfile"
     assert ctx.images["image_a"].build_args == ["--arg1", "val1", "--arg2=val2"]
+    assert ctx.images["image_a"].env == {"SECRET_ENV": "secret:key"}
+    assert ctx.images["image_a"].volumes == ["secret:key:/var/secret/key.txt:ro"]
```

Comment:
`neuro` cli client do not support `ro` spec for secret volumes:
https://github.com/neuromation/platform-client-python/blob/7a0099955c96960ee5d93c12cf488b2f021720f3/neuromation/api/parser.py#L82-L90
Not sure how to fix this issue, options are:
- strip the `:ro` suffix in neuro-flow
- add support of `:ro` suffix in neuro
- disallow access to `volumes` context from images and make user enter a valid string himself.

Reply:
You're right! thank you

Reply:
hmm, it seems like a bug in `neuro-flow`: it fails when mounting secrets as volumes as well:
```
kind: live
title: Example config

volumes:
  volume_sec:
    remote: secret:wandb
    mount: /var/secret.txt
    read_only: True

jobs:
  test_sec_vol_ref_rw:
    image: ubuntu
    preset: cpu-small
    cmd: cat /var/secret.txt
    volumes:
      - ${{ volumes.volume_sec.ref_rw }}

  test_sec_vol_empty:
    image: ubuntu
    preset: cpu-small
    cmd: cat /var/secret.txt
    volumes:
      - ${{ volumes.volume_sec }}
```
Both fail:
```
$ nf run test_sec_vol_ref_rw
ERROR: Invalid secret file specification 'secret:wandb:/var/secret.txt:rw'

$ nf run test_sec_vol_empty
ERROR: Invalid volume specification 'VolumeCtx(id='volume_sec', remote=URL('secret:wandb'), mount=PurePosixPath('/var/secret.txt'), read_only=True, local=None, full_local_path=None)'
```

Reply:
I think, first we should fix secrets mount into the job, and then return back to current PR.

Reply:
I am going to patch `neuro` to allow optional postfix `:ro`/`:rw` for secret volumes:
```
$ neuro run ... -v secret:key:/var/secret.txt  # valid
$ neuro run ... -v secret:key:/var/secret.txt:ro  # valid
$ neuro run ... -v secret:key:/var/secret.txt:rw  # raises exception
```
@asvetlov WDYT?

Reply:
as @asvetlov advised, we will support only explicit hard-coded values in image's `volumes` definition:
```
jobs:
  test_sec:
    image: ubuntu
    preset: cpu-small
    cmd: cat /var/secret.txt
    volumes:
      - secret:wandb:/var/secret.txt
```